kernel-commits.mk

```diff
@@ -1,6 +1,6 @@
 KERNEL_COMMIT_amd64_v5.10.186_generic = 37256d6aff33
 KERNEL_COMMIT_amd64_v6.1.38_generic = ddbb9e41fa01
-KERNEL_COMMIT_amd64_v6.1.38_rt = 94c2c02bf1b9
+KERNEL_COMMIT_amd64_v6.1.38_rt = 1ead4c5f5ee7
 KERNEL_COMMIT_amd64_v6.1.68_generic = b4d7ad0a73b1
 KERNEL_COMMIT_arm64_v5.10.104_nvidia = 11760a953d2d
```

Comment:
@rouming , may I ask you to take this opportunity and also update NVIDIA's kernel?

HASH is 69009e88617e

```
eve-kernel-arm64-v5.10.104-nvidia:
69009e88617e Dockerfile.gcc: Change rtw88 revision
```

Reply:
Thanks a lot!